e2e/cypress/integration/messaging/group_message_spec.js

```diff
@@ -79,4 +90,196 @@ describe('Group Messages', () => {
+    });
+
+    it('MM-T465 - Assert that group message participant sees', () => {
+        // # Create a group message with two other users
+        const participants = users.slice(0, 2);
+        createGroupMessageWith(participants);
+        cy.wait(TIMEOUTS.HALF_SEC);
+
+        const sortedParticipants = participants.sort((a, b) => {
+            return a.username > b.username ? 1 : -1;
+        });
+
+        // * Assert that intro message includes the right copy
+        const expectedChannelInfo = `This is the start of your group message history with ${sortedParticipants[0].username}, ${sortedParticipants[1].username}.Messages and files shared here are not shown to people outside this area.`;
+        cy.get('#channelIntro p.channel-intro-text').first().should('contain', expectedChannelInfo);
+        cy.get('#channelIntro .profile-icon').should('have.length', '2');
+
+        cy.location().then((loc) => {
+            const channelId = loc.pathname.split('/').slice(-1)[0];
+
+            // * Assert that sidebar displays the group channel
+            cy.get(`#sidebarItem_${channelId}`).should('contain', `${sortedParticipants[0].username}, ${sortedParticipants[1].username}`);
+
+            // * Assert that member count shows next to the channel name
+            cy.get(`#sidebarItem_${channelId} .status`).eq(0).should('contain', '2');
+        });
+    });
+
+    it('MM-T469 - Post an @mention on a group channel', () => {
+        spyNotificationAs('withNotification', 'granted');
+
+        // # Create a group message with two other users
+        const participants = users.slice(0, 2);
+        createGroupMessageWith(participants);
+        cy.wait(TIMEOUTS.HALF_SEC);
+
+        // # Post a message as a different user
+        cy.getCurrentChannelId().then((channelId) => {
+            cy.apiLogin(participants[0]);
+
+            cy.postMessageAs(participants[0], `@${testUser.username} Hello!!!`, channelId);
+
+            // * Assert that user receives notification
+            cy.wait(TIMEOUTS.HALF_SEC);
+            cy.get('@withNotification').should('have.been.called');
+        });
+    });
+
+    it('MM-T475 - Channel preferences, mute channel', () => {
+        spyNotificationAs('withNotification', 'granted');
+
+        // # Create a group message with two other users
+        const participants = users.slice(0, 2);
+        createGroupMessageWith(participants);
+        cy.wait(TIMEOUTS.HALF_SEC);
+
+        // # Clicks on Mute Channel
+        cy.get('#channelHeaderDropdownButton button').click().then(() => {
+            cy.get('#channelToggleMuteChannel button').click().then(() => {
+                // * Assert that channel is muted
+                cy.get('#toggleMute').should('be.visible');
+            });
+        });
+
+        // # Post a message as a different user
+        cy.getCurrentChannelId().then((channelId) => {
+            cy.apiLogin(participants[0]);
```

Comment:
Session should remain as `testUser`
```suggestion
```

Reply:
amended in https://github.com/mattermost/mattermost-webapp/pull/7181/commits/82a5219d44341f5ae5a21b0da5fde24545eab1b3